tests/develop.py

```diff
@@ -1,15 +1,9 @@
 """Create a development and testing environment using a virtualenv."""
-
-from __future__ import unicode_literals
```

Comment:
This can be automated with [`pyupgrade --py37-plus **/*.py`](https://github.com/asottile/pyupgrade#__future__-import-removal)